src/main/java/taskat4/Human.java

```diff
@@ -0,0 +1,91 @@
+package taskat4;
+
+import java.io.IOException;
+
+public abstract class Human {
+
+    private final boolean gender;
+    private String name;
+    private String surname;
+    private float height;
+    private float weight;
+
+    public Human(boolean gender, String name, String surname, float height, float weight) {
+        this.gender = gender;
+        this.name = name;
+        this.surname = surname;
+        this.height = height;
+        this.weight = weight;
+    }
+
+    public String toString(boolean bo) {
+        String str;
+        if (bo) str = "Male";
+        else str = "Female";
+        return ("This human " + name + " and surname " + surname + ", is " + height + " cm tall and " + weight + "weight kilo is " + str);
+    }
+
+    public boolean beTogether(float ek1, float ek2) {
```

Comment:
what is that names of input parameters stands for?

Reply:
and why this method takes float as an input parameter?
this method should take only 1 argument, you should compare object that would call this method to the object passed as a parameter

Reply:
its named like "ekzemplyar". Bad naming